--- setup.py	2010-12-11 23:05:25.000000000 +0800
+++ setup.py.new	2010-12-23 12:20:49.392803775 +0800
@@ -8,5 +8,2 @@
 
-import docutils.core
-import docutils.writers.manpage
-
 from distutils.core import setup
@@ -78,23 +75,4 @@
 
-class build_man(NoOptionCommand):
-    description = 'Generate and compress man page'
-
-    def run(self):
-        self.mkpath('build')
-
-        sourcefile = 'data/arandr.1.txt'
-        gzfile = os.path.join('build', 'arandr.1.gz')
-
-        if newer(sourcefile, gzfile):
-            rst_source = open(sourcefile).read()
-            manpage = docutils.core.publish_string(rst_source, writer=docutils.writers.manpage.Writer())
-            info('compressing man page to %s', gzfile)
-
-            if not self.dry_run:
-                compressed = gzip.open(gzfile, 'w', 9)
-                compressed.write(manpage)
-                compressed.close()
-
 class build(_build):
-    sub_commands = _build.sub_commands + [('build_trans', None), ('build_man', None)]
+    sub_commands = _build.sub_commands + [('build_trans', None)]
     def run(self):
@@ -150,3 +128,2 @@
             'build_trans': build_trans,
-            'build_man': build_man,
             'build': build,
@@ -160,3 +137,2 @@
             ('share/applications', ['data/arandr.desktop']), # FIXME: use desktop-file-install?
-            ('share/man/man1', ['build/arandr.1.gz']),
             ],
